Sync Client: could not find `runtime` in `tokio`

Hello,

Thank you for this nice project :-).

Trying the small project with tokio-modbus 0.5.0 raises a compilation error:

```toml
[dependencies]
tokio-modbus = { version = "0.5", default-features = false, features = ["tcp", "sync"] }
```

```rs
pub fn main() {
    use tokio_modbus::prelude::*;

    let socket_addr = "192.168.0.222:502".parse().unwrap();
    let mut ctx = client::sync::tcp::connect(socket_addr).unwrap();
    let buff = ctx.read_input_registers(0x1000, 7).unwrap();
    println!("Response is '{:?}'", buff);
}
```

It raises the following error:

```sh
cargo run
   Compiling tokio-modbus v0.5.0
error[E0433]: failed to resolve: could not find `runtime` in `tokio`
  --> /Users/hwhost/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-modbus-0.5.0/src/client/sync/mod.rs:50:18
   |
50 |     core: tokio::runtime::Runtime,
   |                  ^^^^^^^ could not find `runtime` in `tokio`

error[E0433]: failed to resolve: could not find `runtime` in `tokio`
  --> /Users/hwhost/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-modbus-0.5.0/src/client/sync/tcp.rs:16:30
   |
16 |     let rt = tokio::runtime::Builder::new_current_thread()
   |                              ^^^^^^^ not found in `tokio::runtime`
   |
help: consider importing one of these items
   |
1  | use std::thread::Builder;
   |
1  | use tokio_util::codec::length_delimited::Builder;
   |

For more information about this error, try `rustc --explain E0433`.
error: could not compile `tokio-modbus` due to 2 previous errors
```

Can you reproduce this error? It seems to happen only with the `sync` feature.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -36,7 +36,7 @@ tokio = { version = "1", features = ["net", "macros", "io-util", "rt", "time"] }
 default = ["tcp", "rtu"]
 rtu = ["tokio-serial", "futures-util/sink"]
 tcp = ["tokio/net", "futures-util/sink"]
-sync = []
+sync = ["tokio/rt"]
 server = ["futures", "socket2/all", "tokio/rt", "tokio/rt-multi-thread"]
 tcp-server-unstable = ["server"]
 

